app/javascript/inject.js

```diff
@@ -99,9 +119,9 @@ function injectRatings(node, ratings) {
 				node.appendChild(imdbSpan());
 			}
 		}
-		if (rtRating && !node.querySelector(".rtRating")) {
-			node.appendChild(rtLogoNode());
-			node.appendChild(rtRatingNode(rtRating));
+		if (rtUrl && !node.querySelector(".rtRating")) {
+			node.appendChild(rtLogoNode(rtUrl));
+			if (rtRating) node.appendChild(rtRatingNode(rtUrl, rtRating));
```

Comment:
So now this will only inject the logo and rating if there's a `rtUrl` right? What if there's a rating but no url? `rtLogoNode` already handles if there's no `rtUrl` so can you change this to always append `rtLogoNode` and `rtRatingNode` if there's an `rtRating`? Basically just changing the `if (rtUrl...` back to `if (rtRating...`?

Reply:
Good point! I've decided that `if ((rtRating || rtUrl) &&...` is probably the best approach. 

This not only accounts for the edge case you mention of a rating from OMDB but no rating from IMDB, but it also accounts for the edge case of a RT link to a title existing but no TomatoMeter rating existing. In this case, it will only display a clickable tomato, similar to how IMDB behaves in the absence of a rating.

See the Netflix title "Jack Whitehall: Travels With My Father" for an example of the second edge case I describe.